resources/kernel/make_busybox.sh

```diff
@@ -0,0 +1,213 @@
+#!/bin/bash
+
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR BSD-3-Clause
+
+# This script illustrates the build steps for `vmlinux-hello-busybox`.
+
+set -e
+
+WORKDIR="/tmp/vmlinux_busybox"
+SOURCE=$(readlink -f "$0")
+TEST_RESOURCE_DIR="$(dirname "$SOURCE")"
+
+KERNEL="linux-4.14.176"
+KERNEL_ARCHIVE="$KERNEL.tar.xz"
+KERNEL_URL="https://cdn.kernel.org/pub/linux/kernel/v4.x/$KERNEL_ARCHIVE"
+
+BUSYBOX="busybox-1.26.0"
+BUSYBOX_ARCHIVE="$BUSYBOX.tar.bz2"
+BUSYBOX_URL="https://busybox.net/downloads/$BUSYBOX_ARCHIVE"
+
+# Reset index for cmdline arguments for the following `getopts`.
+OPTIND=1
+# Flag for optionally building a guest that halts.
+HALT=
+# Number of CPUs to use during the kernel build process.
+MAKEPROCS=1
+# Flag for optionally cleaning the workdir and recompiling the kernel.
+CLEAN=
+
+# Supported arguments:
+# * `-h`: build a guest that halts.
+# * `-j`: compile with `make -j` (on all available CPUs).
+#         TODO: pass the value for `-j`.
```

Comment:
Also passing the VMLINUX name as arg could be a good TODO. Looks like if we modify something in the script (and we don't want to recompile the kernel), we have to also change that name, otherwise the old image will be copied (or maybe we could overwrite de old image :-?).
Can we open an issue for future improvements to this script so we don't forget about them?